parsely-parsely-page.php

```diff
@@ -4,6 +4,9 @@
    <script type="application/ld+json">
    <?php echo json_encode($parselyPage); ?>
    </script>
+    <?php if (isset($parselyPage["custom_metadata"])) : ?>
+        <meta name='parsely-metadata' content='<?php echo json_encode($parselyPage['custom_metadata'])?>'>
+        <?php endif; ?>
```

Comment:
yeah, exactly. Documentation is a good idea: it's a bog-standard filter, but we should at least point out it's an option, and what arguments it gets called with so people don't have to dig around to find it